src/main/java/io/github/hypixel_api_wrapper/wrapper/player/HypixelPlayer.java

```diff
@@ -105,4 +110,8 @@ public HypixelGuild getGuild() {
         throw new UnsupportedOperationException();
     }
 
-}
+    public HypixelPlayerStats getStats() {
```

Comment:
I know the field's just called `stats` in the API, but we could probably just call it `getGames()` since it only ever contains stats pertaining to minigames on the network

Reply:
Resolved via eb20aded